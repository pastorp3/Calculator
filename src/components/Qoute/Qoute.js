/* eslint-disable max-len, arrow-body-style */

import React from 'react';
import Navbar from '../Navbar';

const Qoute = () => {
  return (
    <main>
      <Navbar />
      <h3>
        Mathematics is not about numbers, equations, computations,
        or algorithms: it is about understanding. â€“William Paul Thurston
      </h3>
    </main>
  );
};

export default Qoute;

/* eslint-enable max-len, arrow-body-style */
