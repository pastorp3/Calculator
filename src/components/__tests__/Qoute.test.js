import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import Qoute from '../Qoute/Qoute';

it('renders correctly', () => {
  const tree = renderer.create(<BrowserRouter><Qoute /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
