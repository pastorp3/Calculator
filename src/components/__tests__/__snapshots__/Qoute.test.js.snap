// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main>
  <div
    className="navbar"
  >
    <h1>
      Math Magicians
    </h1>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="mid-li"
      >
        <a
          href="/app"
          onClick={[Function]}
        >
          Claculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </div>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h3>
</main>
`;
